prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/git/GitUtils.java

```diff
@@ -0,0 +1,39 @@
+package com.redhat.parodos.tasks.git;
+
+import com.redhat.parodos.workflows.work.WorkContext;
+import lombok.Getter;
+
+import static com.redhat.parodos.workflow.context.WorkContextDelegate.getOptionalValueFromRequestParams;
```

Comment:
Pls don't use static import in production code (at most only for constants)

https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements

> Static imports should not be used in production code, but they should be used in test code, especially for things like import static org.assertj.core.api.Assertions.assertThat;.